src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + df.format(getArea()) + " sq. units, radius: "
+                + df.format(radius) + " units, color: "
```

Comment:
Instead of directly concatenating strings, you might consider using `String.format` for better readability of the formatted message.